server.js

```diff
@@ -40,7 +38,7 @@ passport.use(
     {
       clientID: clientID,
       clientSecret: process.env.GITHUB_SECRET,
-      callbackURL: "http://localhost:8080/github/callback",
+      callbackURL: "http://localhost:3000/github/callback",
```

Comment:
Now that we are implementing this fully, we need to remove any references to localhost from our codebase. You should be able just to use relative addresses with no issue for this.

- [ ] Change to relative URLs